PowerUp/app/src/main/res/layout/avatar_room.xml

```diff
@@ -41,77 +42,89 @@
         android:src="@drawable/cloth1" />
 
     <ImageButton
-        android:id="@+id/faceRight"
+        android:id="@+id/eyeLeft"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignRight="@id/face"
-        android:layout_below="@id/face"
-        android:layout_marginTop="@dimen/face_right_margin_top"
-        android:src="@drawable/ic_keyboard_arrow_right_white_18dp" />
+        android:src="@drawable/ic_keyboard_arrow_left_white_18dp"
+        android:layout_below="@+id/eyes"
```

Comment:
You should only be using `@id` instead of `@+id` when referring to an already-existing id